src/StudentNames.java

```diff
@@ -0,0 +1,12 @@
+/**
+ * Created by darionmoore on 11/28/16.
+ */
+public class StudentNames {
+    String firstName = "";
+    String lastName = "";
+    String studentList = new "firstName" + "lastName";
```

Comment:
no idea what this line is doing.  Please fix to something.  Either studentList is an array or it is something else.